MediaBrowser.Model/Extensions/EnumerableExtensions.cs

```diff
@@ -24,24 +24,27 @@ public static IEnumerable<RemoteImageInfo> OrderByLanguageDescending(this IEnume
                 requestedLanguage = "en";
             }
 
-            var isRequestedLanguageEn = string.Equals(requestedLanguage, "en", StringComparison.OrdinalIgnoreCase);
-
             return remoteImageInfos.OrderByDescending(i =>
                 {
+                    // Image priority ordering:
+                    //  - Images that match the requested language
+                    //  - Images with no language
```

Comment:
Why?

Reply:
[As described above](https://github.com/jellyfin/jellyfin/pull/8946#issue-1509297531), the current implementation conflates _English_ with _No Language_. The effect of this is that _some_ movies in my collection (maybe 5%) are automatically assigned a _No Language_ poster, despite TMDb featuring the top  _English_ poster (e.g. with [The Shining](https://www.themoviedb.org/movie/694-the-shining/)).

Yet my French language libraries aren't affected by this and _don't_ select _No Language_ posters. This behavioural inconsistency lead me to find the above code and submit the PR for consistent treatment of _No Language_ posters.